ajax/libs/smart-app-banner/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "smart-app-banner",
+  "filename": "smart-app-banner.js",
+  "description": "Smart banner for Android or iOS",
```

Comment:
Let's use this one as the description: https://github.com/kudago/smart-app-banner/blob/master/bower.json#L8 which is the same with README.
Thank you